api/res/openapi.yml

```diff
@@ -131,6 +125,42 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/ProblemDetails'
+  /apps/{appName}/logs/{serviceName}/:
+    get:
+      summary: Retrieves the logs from stdout/stderr of the specified container.
+      parameters:
+        - $ref: '#/components/parameters/appName'
+        - in: path
+          name: serviceName
+          description: Name of the service
```

Comment:
Shouldn't *appName* be documented here, too?

Reply:
It is. cf. `$ref: '#/components/parameters/appName'`